__tests__/app.test.js

```diff
@@ -90,4 +89,45 @@ describe('GET /api/articles/:article_id', () => {
         })
     });
 
-});
\ No newline at end of file
+});
+
+
+describe('get  /api/articles/:article_id/comments', () => {
+    it('returns status 200 and an array of corresponding comment objects for the given article_id ', () => {
+        return request(app)
+        .get("/api/articles/3/comments")
+        .expect(200)
+        .then(({body})=> {
+            expect(Array.isArray(body)).toBe(true);
+            expect(body.length).toBe(2);
+            body.forEach((comment) => {
+                expect(comment).toHaveProperty("comment_id");
+                expect(comment).toHaveProperty("votes");    
+                expect(comment).toHaveProperty("created_at");    
+                expect(comment).toHaveProperty("author");    
+                expect(comment).toHaveProperty("body");    
+                expect(comment).toHaveProperty("article_id");    
+
+            });            
+
+        })
+    });
+    it('Returns 404 status and a message for an article_id that does not correspond to any comments ', () => {
```

Comment:
This is going along the right lines but we need to think about what the response should be if an article (that exists) simply has no associated comments. This is not an error (so should have an appropriate status code) and the client should actually receive the returned data from the query (so we should check for this).  Hint the model will have to distinguish between a couple of of things and behave accordingly

Reply:
As per the Trello task we also need to assert the comments are served with the most recent comments first